sdk/src/store.rs

```diff
@@ -186,6 +186,20 @@ impl Store {
         Ok(claim_label)
     }
 
+    /// Add a new Claim to this Store. [`remote_manifest_path`] will be embedded in the
```

Comment:
May not matter since `Store` isn't part of the public API, but in doc comments the `[]` wrapper should only be used when referring to another item that has its own doc entry (i.e. another public struct or enum, but not an argument to this function). The backtick wrappers are sufficient here.

Reply:
👍